<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>283</width>
      <height>234</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>381</width>
      <height>281</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="loginLabel">
       <property name="minimumSize">
        <size>
         <width>38</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>38</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="maximumSize">
        <size>
         <width>357</width>
         <height>27</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="maximumSize">
        <size>
         <width>357</width>
         <height>27</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="rememberCheckBox">
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Remember me</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forgotPasswordButton">
         <property name="maximumSize">
          <size>
           <width>174</width>
           <height>27</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Forgot password?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="maximumSize">
        <size>
         <width>357</width>
         <height>27</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="registerLabel">
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>39</height>
          </size>
         </property>
         <property name="text">
          <string>Don't have an account?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="registerButton">
         <property name="maximumSize">
          <size>
           <width>174</width>
           <height>27</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
